pilot/pkg/security/authz/builder/testdata/http/allow-empty-rule-in.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: security.istio.io/v1beta1
+apiVersion: security.istio.io/v1
```

Comment:
These don't need to change; current prior art from networking is to use the oldest version everywhere (its strictly better - same schema but more compatible, and requires no noise from changing a bunch of files)